rockin/plugins/drilling_machine/drilling_machine_plugin.cpp

```diff
@@ -36,8 +36,28 @@ class DrillingMachinePlugin: public fawkes::Plugin
         DrillingMachinePlugin(Configuration *config) :
                 Plugin(config)
         {
-            thread_list.push_back(new DrillingMachineThread());
-        }
+          std::string mode = "";
+          int encoder_max = 40;
```

Comment:
This should be a configuration param or a constant. 

The problem I have is that the loop rate is a configuration param, so I can't know which rate the ticks_ are increasing.